.github/fabricbot.json

```diff
@@ -1,3600 +0,0 @@
-            ],
-            "timezoneOffset": -8
-          },
-          {
-            "weekDay": 6,
-            "hours": [
-              1,
-              4,
-              7,
-              10,
-              13,
-              16,
-              19,
-              22
-            ],
-            "timezoneOffset": -8
-          }
-        ],
-        "searchTerms": [
-          {
-            "name": "isClosed",
-            "parameters": {}
-          },
-          {
-            "name": "noActivitySince",
-            "parameters": {
-              "days": 60
-            }
-          },
-          {
-            "name": "isUnlocked",
-            "parameters": {}
-          },
-          {
-            "name": "isIssue",
-            "parameters": {}
-          }
-        ],
-        "taskName": "Lock issues closed without activity for over 60 days",
-        "actions": [
-          {
-            "name": "lockIssue",
-            "parameters": {
-              "reason": "resolved"
-            }
-          }
-        ]
-      },
-      "id": "wWxYItL3hl"
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "EmailCleanser",
-      "subCapability": "EmailCleanser",
-      "version": "1.0",
-      "id": "vlRSU7Suz",
-      "config": {
-        "taskName": "Remove Quoted Email Content "
-      }
-    },
-    {
-      "taskType": "trigger",
-      "capabilityId": "IssueResponder",
-      "subCapability": "IssuesOnlyResponder",
-      "version": "1.0",
-      "id": "NEOXUiMV8",
-      "config": {
-        "conditions": {
-          "operator": "and",
-          "operands": [
-            {
-              "name": "isOpen",
-              "parameters": {}
-            },
-            {
-              "name": "hasLabel",
-              "parameters": {
-                "label": "needs triage :mag:"
-              }
-            },
-            {
-              "name": "labelAdded",
-              "parameters": {
-                "label": "In-PR :rocket:"
-              }
-            },
-            {
-              "name": "isPartOfProject",
-              "parameters": {}
-            },
-            {
-              "operator": "or",
-              "operands": [
-                {
-                  "name": "isOpen",
-                  "parameters": {}
-                },
-                {
-                  "name": "hasLabel",
-                  "parameters": {
-                    "label": "needs triage :mag:"
-                  }
-                },
-                {
-                  "name": "isAssignedToSomeone",
-                  "parameters": {}
-                },
-                {
-                  "name": "isPartOfProject",
-                  "parameters": {}
-                }
-              ]
-            }
-          ]
-        },
-        "eventType": "issue",
-        "eventNames": [
-          "issues",
-          "project_card"
-        ],
-        "taskName": "Remove needs triage label",
```

Comment:
This task didn't get ported over, presumably because it references `isPartOfProject` which is not supported.